File: src/main/frontend/src/model/CargoDetailsDto.ts
import {Cargo} from "./Cargo";

export class CargoDetailsDto {
  public constructor(
      public cargo: Cargo | null,
      public totalRent: number | null,
      public totalReturn: number | null,
      public totalPaid: number | null,
      public balance: number | null,
  ) {
  }
}

File: src/main/java/com/raihanorium/springreact/dto/CargoDetailsDto.java
package com.raihanorium.springreact.dto;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.raihanorium.springreact.model.Cargo;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
@JsonInclude(JsonInclude.Include.NON_NULL)
public class CargoDetailsDto {
    private Cargo cargo;
    private Double totalRent;
    private Double totalReturn;
    private Double totalPaid;
    private Double balance;
}
